docs/run-random-beacon.adoc

```diff
@@ -344,6 +351,28 @@ Trust math, not hardware.
 
 Keep uses the Ethereum Ropsten Testnet.
 
+==== Faucet
+
+The KEEP faucet will will issue a 300k KEEP token grant for the provided Ethereum account.  You can
+use the faucet from your web browser or via a terminal using curl.
+
+Faucet Endpoint: https://us-central1-keep-test-f3e0.cloudfunctions.net/keep-faucet-ropsten
+
+To use the faucet you need to pass your Ethereum account to the faucet endpoint with the parameter
+`?account=<eth-account-address>`.
+
+Curl Example:
+```
+curl https://us-central1-keep-test-f3e0.cloudfunctions.net/keep-faucet-ropsten'?account=0x0eC14BC7cCA82c942Cf276F6BbD0413216dDB2bE'
```

Comment:
The mid-parameter quotes threw me for a loop. Thoughts about wrapping the whole string in a loop?